src/server/ApiRoutes/Hold.js

```diff
@@ -69,6 +72,31 @@ function postHoldAPI(
 }
 
 /**
+ * mapLocationDetails(locations)
+ * The function extracts the details of the delivery locations from the location.js and
+ * locationCodes.js based on the locastion id we get from deliveryLocationsByBarcode API.
+ *
+ * @param {array} locations
+ * @return {array}
+ */
+function mapLocationDetails(locations) {
+  locations.map(loc => {
+    _mapObject(locationCodes, (c) => {
+      if (loc['@id'].replace('loc:', '') === c.delivery_location) {
+        loc.address = (locationDetails[c.location]) ?
```

Comment:
Hmm okay. Hopefully it stays that way then or else you're doing a lot of work to just extract one property. If you get more data for each object, you could potentially use it in the future if need be.